src/app/pages/form-validation/form-validation.component.html

```diff
@@ -88,8 +109,34 @@ <h2>Password Validation</h2>
                         type="password"
                         (keyup)="checkPasswordStrength($event.target.value)"
                     />
+                    <button mat-icon-button matSuffix (click)="toggleNewPasswordVisibility()">
+                        <mat-icon>{{ newPasswordVisible ? 'visibility' : 'visibility_off' }}</mat-icon>
+                    </button>
                 </mat-form-field>
-                <mat-form-field>
+                <mat-list style="margin: -40px 0 16px;">
+                    <mat-list-item>
```

Comment:
Can we add a style rule for these list-items that removes the 48px height so they can come closer together:
![image](https://user-images.githubusercontent.com/29152776/88077994-e861dc00-cb49-11ea-8254-a1d9198d9abb.png)

Reply:
It also looks like we should set the background color to gray[50] like we did on the loading states since we are using cards.